### PR TITLE
fix: increase item_id max lenght from 128 to 255 for peer, staff and studenttraining workflows

### PR DESCRIPTION
**Pull request by vunguyen-dmt**
**TL;DR -** [ A short summary of what this PR does and why ]

The current max length for item_id fields of peerworkflow, staffworkflow and studenttrainingworkflow is 128 characters which are insufficient for course_id with a length greater than 70 characters. This PR alters the max length of item_id from 128 to 255.
[Related forum post](https://discuss.openedx.org/t/report-open-response-edx-ora2-bug-cannot-submit-responses-for-courses-with-ids-greater-than-70-characters/12368/2)

**What changed?**

- Increase item_id max length of peerworkflow, staffworkflow and studenttrainingworkflow from 128 to 255.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

1. Check if the definitions of these fields are varchar(255). openedx.assessment_peerworkflow.item_id, openedx.assessment_staffworkflow.item_id and openedx.assessment_studenttrainingworkflow.item_id.
2. Create a new course with an ID > 70 characters.
3. Create an Open Response component that has Student training and Peer and Staff steps.
4. Submit responses and assess them.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @vunguyen-dmt! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by vunguyen-dmt** [comment]
Hi @pomegranited I messed up my commit a bit. I closed https://github.com/openedx/edx-ora2/pull/2183 and create this new PR.

**Review by pomegranited — APPROVED** [review]
This change works as expected, thank you @vunguyen-dmt !

So that I can create a new release, could you please update the version string from `6.3.0` to `6.4.0` in these files:

* [openassessment/__init__.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L5)
* [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

:+1:

- [x] I tested this on my devstack by running migrations, then using previously-created ORA2 blocks, and creating new ones. Everything works as expected.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A
- [x] <strike>User-facing strings are extracted for translation</strike> N/A

**Comment by vunguyen-dmt** [comment]
@pomegranited I've updated it.

**Comment by pomegranited** [comment]
@vunguyen-dmt Gah.. the dreaded commit lint error! All commits have to follow this format: [OEP-51 Conventional Commits](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html)

This is what caught us with the last PR :( To repair this, you'll have to reword your next-to-last commit, and force-push it up to this branch.  Steps:

```bash
> git rebase -i origin/master  # will show your commits since master:
pick a8a6e8ebc fix: increase item_id max lenght from 128 to 255 for peer, staff and studenttraining workflows
pick ee9a9dbbb increase version to 6.5.0

# Change the second "pick" to "reword", this lets you edit that commit message:
pick a8a6e8ebc fix: increase item_id max lenght from 128 to 255 for peer, staff and studenttraining workflows
reword ee9a9dbbb increase version to 6.5.0

# change the message to something like this:
chore: increase version to 6.5.0

# Replace your previous commit
> git push --force
```

**Comment by pomegranited** [comment]
@vunguyen-dmt Oh no, the dreaded [commit lint](https://github.com/openedx/edx-ora2/actions/runs/8185257577/job/22414663962?pr=2188#step:5:10) error again :( Also, there's a [quality issue](https://github.com/openedx/edx-ora2/actions/runs/8185257513/job/22414663783?pr=2188#step:9:144) with e6af99c783d8832b5907bb680c57ab048f5b0023 -- there needs to be a newline at the end of that file.

So you'll need to repair those commits and force push to replace them in this PR. Sorry about that!

Let me know if you have any trouble with this?

**Comment by vunguyen-dmt** [comment]
@pomegranited I've made some changes, please help to check if there are still any issues.

**Comment by openedx-webhooks** [comment]
@vunguyen-dmt 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-ora2&entry.867055334=https://github.com/openedx/edx-ora2/pull/2188&entry.1484655318=https://github.com/vunguyen-dmt&entry.752974735=2024-03-06+10:25&entry.1917517419=2024-03-08+04:35&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by pomegranited** [comment]
Thank you for your contribution, and for working through the CI issues @vunguyen-dmt !
